src/legacy/core_plugins/vis_default_editor/public/components/agg_group.tsx

```diff
@@ -70,7 +70,7 @@ function DefaultEditorAggGroup({
 }: DefaultEditorAggGroupProps) {
   const groupNameLabel = (search.aggs.aggGroupNamesMap() as any)[groupName];
   // e.g. buckets can have no aggs
-  const schemaNames = getSchemasByGroup(schemas, groupName).map(s => s.name);
```

Comment:
Here `schemas` has already contained schemas by group from `groupName`. So we don't need to get them again.